### DISCUSSION TITLE
Cluster of Birdnet Pi

### DISCUSSION THREAD

**Discussion by HGFestl** [Nachtzuster/BirdNET-Pi#219] [Ideas]
Hello! I apologize if my concern is in the wrong section.
I would like to use BirdNET Pi in three stages. First, several Pi's with solar power should analyze bird calls. In the second step, a LoRa WAN network is to be set up to communicate certain status messages, such as the first arrival of a species. In the third step, I would like to improve the quality of the analysis, for example to recognize different calls for further analysis. Individuals or even breeding can perhaps be recognized via contact calls. Has anyone had any experience with this or is interested in sharing it with me?
Best regards, Hans Guenter

**Comment by unosonic** [comment]
hi, I've no experience with this... that said,
First step is easy, just do it. 2nd step would probably mean to query the databases of detections periodically and do something with it, eg. sending it somewhere, via lora, wifi, cable or whatever connectivity, that's not a birdnet issue. 3rd step would mean to create and train your own birdnet model, I guess. Certainly doable, but quite a different businesss than just using a model in an application..
bests, u
